src/eureka/S5_lightcurve_fitting/models/HSTRampModel.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
+
+from .Model import Model
+from ...lib.readEPF import Parameters
+
+
+class HSTRampModel(Model):
```

Comment:
This will need to be added to the s5_fit.py file. E.g., this is how the `polynomial` gets added to the mix:
https://github.com/kevin218/Eureka/blob/3c1092661415f429c4ea685f898afa09679990d9/src/eureka/S5_lightcurve_fitting/s5_fit.py#L621-L637

Reply:
Also, we should add this model to the differentiable_models folder as well (annoying I know, but using PyMC3 has been super helpful for me at least). I definitely want to try merging the differentiable and non-differentiable models sometime, but it's a bit tricky to make the code fast for emcee,dynesty,lsq while still working for PyMC3. It should be doable, it'll just take a bit of finessing.

Reply:
All new parameters should also be added to the docs page (under "S5 Fitted Parameters" I believe) and documented in the S5 ECF and maybe the EPF template. Useful for me testing the code too as it's not immediately clear what range of values might be reasonable